src/Anexia.BDP.CloudLog/Client.cs

```diff
@@ -15,49 +17,47 @@ namespace Anexia.BDP.CloudLog
     /// </summary>
     public class Client
     {
-        private string brokers = "kafka0401.bdp.anexia-it.com:8443";
-        private string api = "https://api0401.bdp.anexia-it.com";
+        private const string api = "https://api0401.bdp.anexia-it.com";
 
-        private string index;
-        private string token;
-        private string caFile;
-        private string certFile;
-        private string keyFile;
-        private string keyPassword;
-        private string clientType;
-        private bool isHttp;
+        private static JsonSerializerOptions _jsonSerializerOptions = new()
+        {
+            PropertyNameCaseInsensitive = true
+        };
+
+        private readonly string _index;
+        private readonly string _token;
+        private readonly bool _isHttp;
         private long queue = 0;
 
-        private Producer<string, string> producer;
+        private string ClientType;
+
         private HttpClient httpClient;
 
+        private readonly Func<long> createTimeStamp =
```

Comment:
Either keep this an lambda expression and rename to `_createTimeStamp` or convert it to a proper method and rename to `CreateTimeStamp`.